README.md

```diff
@@ -73,7 +73,10 @@ problem, or that there might be a better way of writing the code. Therefore, the
 
 # iOS Platform
 
-🚧 Under Construction...
+ Name | Detailed Description
+---|---
+ *Power* |  
+ Save Mode Awareness | Use device API to check if low power mode is enabled, and adapt resources usage accordingly. For example, you can reduce frequency of data update if low power mode is enabled. Your app can query the `ProcessInfo.processInfo.isLowPowerModeEnabled` property at any time to determine whether Low Power Mode is active. Your app can also register to receive notifications when the Low Power Mode state of a device changes.
```

Comment:
I would add the notification name at the end of the sentence, like so:

```suggestion
 Save Mode Awareness | Use device API to check if low power mode is enabled, and adapt resources usage accordingly. For example, you can reduce frequency of data update if low power mode is enabled. Your app can query the `ProcessInfo.processInfo.isLowPowerModeEnabled` property at any time to determine whether Low Power Mode is active. Your app can also register to receive notifications when the Low Power Mode state of a device changes, using `NSProcessInfoPowerStateDidChange`.
```

What do you think?

Documentation: https://developer.apple.com/documentation/foundation/nsnotification/name/1617322-nsprocessinfopowerstatedidchange